src/serve_rendered.js

```diff
@@ -1414,82 +1465,142 @@ export const serve_rendered = {
 
     const queue = [];
     for (const name of Object.keys(styleJSON.sources)) {
+      let source_type;
       let source = styleJSON.sources[name];
       const url = source.url;
 
-      if (url && url.lastIndexOf('mbtiles:', 0) === 0) {
+      if (
+        url &&
+        (url.lastIndexOf('pmtiles:', 0) === 0 ||
```

Comment:
you can use `url.startsWith('pmtiles:')` here

Reply:
updated